app/mapping/TrustDetailsExtractor.scala

```diff
@@ -44,29 +47,49 @@ class TrustDetailsExtractor extends Logging {
     }
   }
 
+  private def extractIfTaxable(answers: UserAnswers)(block: Try[UserAnswers]): Try[UserAnswers] = {
+    if (answers.isTrustTaxable) {
+      block
+    } else {
+      Success(answers)
+    }
+  }
```

Comment:
Could we stick this somewhere else to remove the duplication between this and the PlaybackExtractor? Perhaps in an Object or separate trait?